docs/technical-design/howto-migrations.md

```diff
@@ -0,0 +1,48 @@
+# How to complete migrations
+
+## Background
+Migrations are a controlled way to change existing records in the database or else change the shape of tables/relationships. They are necessary when fields are added, removed, or when the meaning behind fields and their relationships change.
+
+## General Guidance
+- Prioritize testing in environments that have data similar to production. If you are testing on a review app, you should pre-populate data through using the deployed app a user or via Cypress runs to have a diversity of test data available.
+- Run in all environments, in as close to a way as they run in CI.
+- Write migrations that can be tested repeatedly.
```

Comment:
```suggestion
- Write migrations that can be tested repeatedly.  Include logic to skip or handle data that has already been changed or doesn't need to be migrated, so that repeated runs of the lambda don't result in bad data.
```